python/cudf/cudf/core/column/lists.py

```diff
@@ -317,6 +318,39 @@ def from_sequences(
         )
         return res
 
+    def as_string_column(
+        self, dtype: Dtype, format=None, **kwargs
+    ) -> "cudf.core.column.StringColumn":
+        """
+        Create a strings column from a list column
+        """
+        # Convert the leaf child column to strings column
+        cc: List[ListColumn] = []
+        c: ColumnBase = self
+        while isinstance(c, ListColumn):
+            cc.insert(0, c)
+            c = c.children[1]
+        s = c.as_string_column(dtype)
+
+        # Rebuild the list column replacing just the leaf child
+        lc = s
+        for c in cc:
+            o = c.children[0]
+            lc = cudf.core.column.ListColumn(  # type: ignore
```

Comment:
I'll take a look at this at a later point. No need to try and address it in this PR.